### PR TITLE
Update RecycleBinThemes.ps1

### PR DESCRIPTION
**Pull request by technoluc**

### CODE REVIEW

File: RecycleBinThemes.ps1 [hunk 1]
```diff
@@ -3,9 +3,9 @@
 #
 
 $version = "v1.0.0"
-$pictures_path = [Environment]::GetFolderPath("MyPictures")
+#$pictures_path = [Environment]::GetFolderPath("MyPictures")
 # This is where we will store the icons
-$recycle_bin_themes_path = "$pictures_path\RecycleBinThemes"
+$recycle_bin_themes_path = "$env:userprofile\Pictures\RecycleBinThemes"
```
Comment (sdushantha) [inline RecycleBinThemes.ps1:8]: Not sure how correct this is. What would happen if a user uses another language and does not have a directory called Picture? Or maybe they renamed it to "pics"

**Comment by sdushantha** [comment]
From #9:

> Probably my fault for not mentioning, im running windows on parallels. I just got to the same idea as well;
it sets the path as \\Mac\Home\Pictures\RecycleBinThemes\patrick-star-empty.ico,0 

This is quite interesting. If I remember correctly Parallels works differently compared to VMware and VirtualBox as you use the same file system as the host (macOS for example). Which means `\Mac\Home\Pictures\RecycleBinThemes\patrick-star-empty.ico` is actually on your macOS system and can be accessed using Finder right?

File: RecycleBinThemes.ps1 (hunk 1 above)
Comment (technoluc) [reply RecycleBinThemes.ps1:8]: I've checked it on a Dutch (my native language) W11 Virtual Machine I have running and even though I've opened the folder "Afbeeldingen" which is Dutch for pictures the file paths are in English. I don't know if that's for every language. image

**Comment by technoluc** [comment]
Yes but it is in fact not \Mac but \\Mac so I think maybe windows sees that as a network share or something? Idk it's just not working

File: RecycleBinThemes.ps1 (hunk 1 above)
Comment (technoluc) [reply RecycleBinThemes.ps1:8]: also, when I rename the folder to pics the path is still \Pictures

**Comment by sdushantha** [comment]
Can you split this PR into two seperate one? One for the new icon and one for ``$env:userprofile\Pictures`. This is so that I can keep things organized

**Comment by technoluc** [comment]
Allright i opened two new PR’s, one for the patch and one for the dachshund. New to GitHub so was a bit messy

**Pull request closed without merging**